### ISSUE TITLE
Turn an example from robotics into a tutorial?

### ISSUE THREAD

**Issue by ChrisRackauckas** [SciML/DiffEqFlux.jl#477]
https://arxiv.org/abs/2012.06684

@samuela is there code to share that could become a tutorial? I think it would be good to make one.

**Comment by samuela** [comment]
Yeah, absolutely! I'd love to contribute a tutorial! Most of the code is actually open source at https://github.com/samuela/research but I plan to do a proper code release soon that is more approachable for others.

Do you have in mind a section of the docs for the tutorial? I'd be happy to put something together once I wrap up some final changes to the paper...

**Comment by ChrisRackauckas** [comment]
I'd say put it together there. We can move it around once it exists. I want to get them tested and tracked so they stay as an updated reference.

**Comment by aditya-2020** [comment]
Sir, is this issue open. If yes, can I give this a try as my first issue.

**Comment by ChrisRackauckas** [comment]
It's open.
